src/form-data-wrapper.js

```diff
@@ -124,15 +124,64 @@ export class PolyfillFormDataWrapper {
 }
 
 /**
- * Wrap the native FormData implementation.
+ * Wrap the native `FormData` implementation.
  *
+ * NOTE: This differs from the standard `FormData` constructor. This constructor
+ * includes a submit button if it was used to submit the `opt_form`, where
+ * the native `FormData` constructor does not include the submit button used to
+ * submit the form.
+ * {@link https://xhr.spec.whatwg.org/#dom-formdata}
  * @implements {FormDataWrapperInterface}
  */
 class NativeFormDataWrapper {
   /** @override */
   constructor(opt_form) {
     /** @private @const {!FormData} */
     this.formData_ = new FormData(opt_form);
+
+    this.maybeIncludeSubmitButton_(opt_form);
+  }
+
+  /**
+   * If a submit button is focused (because it was used to submit the form),
+   * add its name and value to the `FormData`, since publishers expect the
+   * submit button to be present.
+   * @param {!HTMLFormElement=} opt_form
+   * @private
+   */
+  maybeIncludeSubmitButton_(opt_form) {
+    // If a form is not passed to the constructor,
+    // we are not in a submitting code path.
+    if (!opt_form) {
+      return;
+    }
+
+    // If the focused element is not a submit button or does not have a name,
+    // then it does not need to be included in the `FormData`
```

Comment:
I do not think this is correct. If the user does not submit the form via a submit button then the browser will include the _first_ named submit button in the form data. This is an implicit submit button press.

Reply:
You're right, I'll make the change. Thanks